spec/rester/service/middleware/request_handler_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'support/test_service'
+
+module Rester
+  module Service::Middleware
+    RSpec.describe RequestHandler do
+      let(:service) {
+        Class.new(TestService).tap { |klass|
+          allow(klass).to receive(:name) { 'TestService' }
+        }
+      }
```

Comment:
This should be possible with `RequestHandler.new(service.new)`.  `Service#call` doesn't build the full middleware stack, `Service#rack_call` does.